### ISSUE TITLE
Readme  quick install unclear regarding included idP

### ISSUE THREAD

**Issue by xpufx** [netbirdio/netbird#2956]
Would you please add a little blurb in the quick install with docker section making it clear that "an identity provider - zitadel - will be installed". I thought I could follow the quick install docs, THEN look at the advanced install with another idP section.

**Comment by Marcus1Pierce** [comment]
As far as I know, all the quick install tutorials from [netbird docs](https://docs.netbird.io/selfhosted/selfhosted-quickstart#download-and-run-the-script) and [github readme](https://github.com/netbirdio/netbird?tab=readme-ov-file#quickstart-with-self-hosted-netbird) all of them use the link https://github.com/netbirdio/netbird/releases/latest/download/getting-started-with-zitadel.sh which contains the file name `getting-started-with-zitadel.sh`

**Comment by xpufx** [comment]
Yes they do. That's even more confusing. Maybe I am daft but I was wondering why when I was trying to install netbird, i was using a getting started with zitadel script. I thought maybe netbird was called zitadel before and it was an old link.

**Comment by mgarces** [comment]
We can try to clarify our documentation, thank your for this. On our docs we have two sections "Get Started" and "Self-Hosted NetBird", and they both contain the submenu "Getting Started"; do you think this also causes confusion @xpufx?

**Comment by nazarewk** [comment]
Hello @xpufx,

We're currently reviewing our open issues and would like to verify if this problem still exists in the [latest NetBird version](https://github.com/netbirdio/netbird/releases).

Could you please confirm if the issue is still there?

We may close this issue temporarily if we don't hear back from you within **2 weeks**, but feel free to reopen it with updated information.

Thanks for your contribution to improving the project!

**Comment by xpufx** [comment]
Thank you.
